Mathlib/GroupTheory/SpecificGroups/BraidGroup/Basic.lean

```diff
@@ -0,0 +1,82 @@
+/-
+Copyright (c) 2024 Hannah Fechtner. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Hannah Fechtner
+-/
+import Mathlib.GroupTheory.PresentedGroup
+import Mathlib.Data.Nat.Dist
+
+/-!
+#  Braid Groups
+
+Artin's braid group on infintely many strands
+
+## Main definitions
+
+* `BraidGroupInf ` is the braid group on infinitely many strands, defined as a PresentedGroup
+  on the braid relations
+
+* `braid_group n` is the braid group on n+1 strands (thus n generators) - coming in the next PR
+
+## Main results
+* none in this PR
+
+
+## Tags
+braid group
+
+## TODO
+
+-/
+
+namespace BraidGroup
+
+/-- the set of relations on FreeGroup ℕ corresponding to vaild braid moves -/
+def braid_rels_inf : Set (FreeGroup ℕ) :=
+  { r | ∃ i j : ℕ , 1 = Nat.dist i j ∧ r = .of i * .of j * .of i *
+    (.of j)⁻¹ * (.of i)⁻¹ * (.of j)⁻¹} ∪
+  { r | ∃ i j : ℕ, 2 ≤ Nat.dist i j ∧ r = .of i * .of j * (.of i)⁻¹ * (.of j)⁻¹}
+
+/-- Artin's braid group on infinitely many strands -/
+def BraidGroupInf := PresentedGroup braid_rels_inf
```

Comment:
In other files, this group will have the name `BraidGroup.BraidGroupInf`. Is that your intention? I think we should have `BraidGroupInf` and `BraidGroup` as two namespaces. And so this definition should live in the root namespace.